internal/features/rootmodules/root_modules_feature.go

```diff
@@ -117,11 +117,11 @@ func (f *RootModulesFeature) InstalledModuleCalls(modPath string) (map[string]tf
 	return f.Store.InstalledModuleCalls(modPath)
 }
 
-// TerraformVersion tries to find a modules Terraform version on a best effort basis.
+// TofuVersion tries to find a modules Terraform version on a best effort basis.
```

Comment:
```suggestion
// TofuVersion tries to find a modules Tofu version on a best effort basis.
```

Reply:
Can you double check this file please? There are multiple references that could be renamed IINW.